src/main/java/frc/robot/subsystems/drivetrain/SwerveModule.java

```diff
@@ -94,8 +94,8 @@ public SwerveModule(SwerveModuleConfigBase config) {
         driveMotor.selectProfileSlot(1, 0);
         driveMotor.configIntegratedSensorInitializationStrategy(SensorInitializationStrategy.BootToZero);
 
-        driveMotor.configOpenloopRamp(Constants.SwerveModule.RAMP_RATE, Constants.TALON_TIMEOUT);
-        driveMotor.configClosedloopRamp(Constants.SwerveModule.RAMP_RATE, Constants.TALON_TIMEOUT);
+        driveMotor.configOpenloopRamp(0, Constants.TALON_TIMEOUT);
+        driveMotor.configClosedloopRamp(0, Constants.TALON_TIMEOUT);
```

Comment:
Don't do that, just create 2 functions that let you choose the mode (motor-intenal or code defined)